source/_integrations/fritzbox.markdown

```diff
@@ -54,6 +54,7 @@ Additionally, we also support to trigger smarthome templates.
 - [FRITZ!DECT 500](https://en.avm.de/products/fritzdect/fritzdect-500/)
 - [Eurotronic Comet DECT](https://eurotronic.org/produkte/dect-ule-heizkoerperthermostat/comet-dect/)
 - [Magenta SmartHome LED E27](https://www.smarthome.de/geraete/smarthome-led-lampe-e27-farbig-weiss)
+- Magenta SmartHome LED E27 warmwhite
```

Comment:
```suggestion
- [Magenta SmartHome LED E27 warmwhite](https://www.smarthome.de/geraete/smarthome-wlan-led-lampe-e27-weiss)
```

Reply:
also please adjust the name above to indicate it is the colored bulb

Reply:
I added the color info to the existing tested bulb.

Unfortunately they don't seem to sell the warmwhite lamp anymore. The one i have uses the DECT-ULE HANFUN protocol for communicaiton, not wifi. I also couldn't find any other link.